core/src/avm2/value.rs

```diff
@@ -34,8 +34,6 @@ pub enum Hint {
 }
 
 /// An AVM2 value.
-///
-/// TODO: AVM2 also needs Scope, Namespace, and XML values.
```

Comment:
this comment isn't _entirely_ irrelevant, as in: IIRC FP does have a primitive type for namespaces. I don't really understand _why_ or if it can ever matter, so yeah let's drop it :)